aws-predictions-tfl/src/main/java/com/amplifyframework/predictions/adapter/SentimentTypeAdapter.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.adapter;
+
+import androidx.annotation.NonNull;
+
+import com.amplifyframework.predictions.models.SentimentType;
+
+import java.util.Locale;
+
+/**
+ * Utility to convert third-party {@link SentimentType} equivalent
+ * into Amplify-compatible data structure.
+ */
+public final class SentimentTypeAdapter {
+    @SuppressWarnings("checkstyle:all") private SentimentTypeAdapter() {}
+
+    /**
+     * Converts the sentiment string returned by Tensorflow Lite
+     * Interpreter into a format supported by Amplify Predictions.
+     * @param sentiment Sentiment type returned by AWS Comprehend
+     * @return Amplify's {@link SentimentType} enum
+     */
+    @NonNull
+    public static SentimentType fromTensorflow(@NonNull String sentiment) {
+        switch (sentiment.toLowerCase(Locale.US)) {
+            case "positive":
+                return SentimentType.POSITIVE;
+            case "negative":
+                return SentimentType.NEGATIVE;
+            default:
+                return SentimentType.UNKNOWN;
```

Comment:
In software, there's always a choice, between "Fail fast" and "degrade gracefully."

With exception handling, sometimes we see this as "bubble up the error," versus. "return null."

With object construction - here, this enum factory - we have the same sort of thing to consider.  We can either:
1. Throw an error if the input is unknown/invalid. Perhaps, `InvalidArgumentException`.
2. Degrade to an `UNKNOWN`, type, like you are.

I think the key thing to consider is this: Is the system useful/usable when the value is `UNKNOWN`? If not, maybe we should "fail fast."

Reply:
I think it is useful to return unknown if the model decided that it ran its course successfully and was ultimately unable to determine the specific value for a feature. It will serve as a "value" of feature itself.

Reply:
Ah okay, yes! I agree. This is an important distinction. Unknown as an output of the prediction, versus unknown due to not having results from a prediction.